RetoEuskalmet/src/main/java/modelo/DescargarDatosMetereologicos.java

```diff
@@ -1,14 +1,17 @@
 package modelo;
 
+import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.util.Iterator;
 
 public class DescargarDatosMetereologicos {
 
-	public void DescargarDatosMetereologicos() throws FileNotFoundException, IOException {
-		
+	public Boolean DescargarDatosMetereologicos() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.